README.md

```diff
@@ -258,30 +258,30 @@ Standard release channel toolchain names have the following form:
 <host>          = <target-triple>
 ```
 
-'channel' is either a named release channel or an explicit version
-number, such as "1.8.0". Channel names can be optionally appended with
-an archive date, as in 'nightly-2014-12-18', in which case the
+`channel` is either a named release channel or an explicit version
+number, such as `1.8.0`. Channel names can be optionally appended with
+an archive date, as in `nightly-2014-12-18`, in which case the
```

Comment:
I think this change is incorrect - the quoted text are quotes, not code.

Reply:
I found it's easier to read by using backtick. If you have strong objection with it, I will change it back.